### PR TITLE
[OpenBLAS] Set common target for binaries to Power8

### PR DESCRIPTION
**Pull request by vchuravy**

### CODE REVIEW

**Comment by vchuravy** [comment]
Locally this leads to:

```
 OpenBLAS build complete. (BLAS CBLAS LAPACK LAPACKE)

  OS               ... Linux             
  Architecture     ... power               
  BINARY           ... 64bit                 
  Use 64 bits int    (equivalent to "-i8" in Fortran)      
  C compiler       ... GCC  (cmd & version : gcc (GCC) 8.3.1 20191121 (Red Hat 8.3.1-5))
  Fortran compiler ... GFORTRAN  (cmd & version : GNU Fortran (GCC) 8.3.1 20191121 (Red Hat 8.3.1-5))
  Library Name     ... libopenblas64_p-r0.3.21.a (Multi-threading; Max num-threads is 512)
  Supporting multiple power cpu models with minimum requirement for the common code being POWER8
```

File: O/OpenBLAS/common.jl [hunk 1]
```diff
@@ -147,7 +147,7 @@ function openblas_script(;num_64bit_threads::Integer=32, openblas32::Bool=false,
     elif [[ ${target} == arm-* ]]; then
         flags+=(TARGET=ARMV7)
     elif [[ ${target} == powerpc64le-* ]]; then
-        flags+=(TARGET=GENERIC DYNAMIC_ARCH=1)
+        flags+=(TARGET=Power8 DYNAMIC_ARCH=1)
```
Comment (vchuravy) [inline O/OpenBLAS/common.jl:150]:
```suggestion
        flags+=(TARGET=POWER8 DYNAMIC_ARCH=1)
```

**Comment by giordano** [comment]
Good to go?

**Comment by vchuravy** [comment]
Yeah intentionally not triggering a new build.

**Pull request merged**